src/codegate/pipeline/sensitive_data/manager.py

```diff
@@ -0,0 +1,59 @@
+import json
+from typing import Dict, Optional
+import structlog
+from codegate.pipeline.sensitive_data.session_store import SessionStore
+
+logger = structlog.get_logger("codegate")
+
+
+class SensitiveData:
+    """Represents sensitive data with additional metadata."""
+
+    def __init__(self, original: str, service: Optional[str], type: Optional[str]):
```

Comment:
it might be nice to have an enum with the services instead of copying strings over. Or a constant in the modules if we want to keep a string (e.g. SENSITIVE_DATA_SERVICE="pii" in the PII pipeline and SENSITIVE_DATA_SERVICE="secrets" in the secrets pipeline)

Reply:
Oh sorry I just re-read the rest and it seems like we are using a string in the secrets module that corresponds to the secret kind